src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -165,7 +166,7 @@ private async Task CheckLoginRequestStatus()
 
         private async Task CreatePasswordlessLoginAsync()
         {
-            await _deviceActionService.ShowLoadingAsync(AppResources.Loading);
+            await Device.InvokeOnMainThreadAsync(async () => await _deviceActionService.ShowLoadingAsync(AppResources.Loading));
```

Comment:
No need for `async/await` inside.
Also don't you need invoking on main thread when setting the `FingerprintPhrase`, are you getting the property changed on the view?

Reply:
Yes the Fingerprint Phrase is updating and working fine.